docs/Upgrade-Guide.md

```diff
@@ -1,14 +1,37 @@
+## Table of Contents
```

Comment:
I still like the title to be above the TOC, so the user knows what page they are on